snappi_ixnetwork/trafficitem.py

```diff
@@ -610,7 +610,7 @@ def results(self, request):
         """Return flow results
         """
         # setup parameters
-        self._column_names =  request._properties.get('column_names')
+        self._column_names = request._properties.get('column_names')
```

Comment:
_properties is really require. We can access that through request.column_names

Reply:
true but if the column_names are not set from the script then request.column_names will create an instance (as it is @property decorator), to avoid used _properties

Reply:
@alakendu @Rangababu-R let's address this in a separate PR. for now it's not that harmful

Reply:
Yes this is just a concept. You can take any one. 

@Rangababu-R  I feel that will create because it set within constructor
```python
    def __init__(self, parent=None, choice=None, flow_names=None, column_names=None, metric_group_names=None):
        super(FlowMetricsRequest, self).__init__()
        self._parent = parent
        self._choice = choice
        self._set_property('flow_names', flow_names)
        self._set_property('column_names', column_names)
        self._set_property('metric_group_names', metric_group_names)
```